services/ux-backend/main.go

```diff
@@ -61,9 +78,19 @@ func main() {
 		klog.Info("shutting down!")
 		os.Exit(-1)
```

Comment:
Let's replace all instances of os.exit with klog.Exit. If you disagree, it's okay.

Reply:
I agree, but unless Leela has introduced new usages of `os.Exit`, this change shouldn't be introduced into this specific PR

Reply:
yes, not edited. resolving thread.